assets/js/base/components/disclosure-widget/index.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * External dependencies
+ */
+import { useState } from '@wordpress/element';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import { Icon, chevronUp, chevronDown } from '@woocommerce/icons';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+const DisclosureWidget = ( {
```

Comment:
I'm not sure on the name of this component. _Disclosure_ connotes something privacy or terms and conditions related, and _Widget_ suggests a generic action tool.

Maybe `InformationPanel` or `DescriptionPanel`? Or even just `Panel` (while realizing the potential confusion with the WordPress component, I think if we're dropping usage of that anyways, this shouldn't be that much of an issue).

Reply:
Sounds good. Renamed to `Panel` in 92be5de5f52c4bc6453c893b00faa141b47a227c.